app/models/authentication_method.rb

```diff
@@ -20,6 +24,8 @@ def set_one_time_password_secret
     self.one_time_password_secret ||= ROTP::Base32.random
   end
 
+  # @todo This is dangerous because we're creating people incidentally
+  #       instead of intentionally...
```

Comment:
Yeah, this is something to think about. I definitely find it counter-intuitive that in order to create a new `Person` that can log in we are creating an `AuthenticationMethod`. It seems like we would want to know which `Person` is associated with an `AuthenticationMethod`?

It feels more natural that we start with the `Person`. We could have a `Person` API that accepts nested attributes for an `AuthenticationMethod`, and have minimal required attributes for the `Person` (a name? something else?).

Reply:
If we go that way, we probably want to remove the `optional: true` on line 6, and enforce that an `AuthenticationMethod` is always associated with a `Person`.

Reply:
That's a great point! I think for this one, we're going to:

1. leave Person as optional
2. Not fix the `set_person` just yet (but flag it as deprecated and remove it eventually)
3. Probably create a Person endpoint as well, that _does_ accept the Nested Attributes, so that our Tests can create a Person right-off the bat without making two API calls.

The reason we're going "backwards" is because we want to support people "inviting" someone who is not yet a Registered Person in Convene; and once the person sign in with that authentication method, we bind it to the Person who we create on Sign In.